catalog/nsili/catalog-nsili-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -36,6 +36,15 @@
         <AD description="Poll Interval to Check if the Source is available (in minutes - minimum 1)."
             name="Poll Interval" id="pollInterval"
             required="true" type="Integer" default="5"/>
+
+        <AD description="User ID for CORBA Service Access"
```

Comment:
Further up there is this:
<AD description="The Username to be used for authentication with HTTP server."
            name="HTTP/S Username" id="cxfUsername"
            required="false" type="String"/>

So I chose CORBA Service as the description to help differentiate.